File: Python Book/4. Simple Conditions/12_speed_info/speed_info.py
speed = float(input())
if speed <= 10:
    print("slow")
elif 50 >= speed > 10:
    print("average")
elif 150 >= speed > 50:
    print("fast")
elif 1000 >= speed > 150:
    print("ultra fast")
elif speed > 1000:
    print("extremely fast")

File: Full Java Basics/Conditional-Statements-In-Java/SpeedInfo.java
import java.util.Scanner;

public class SpeedInfo {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double speed = sc.nextDouble();
        if (speed <= 30)
            System.out.println("Slow");
        else
            System.out.println("Fast");
    }
}
